index.html

```diff
@@ -1,14 +1,13 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link

```

Comment:
- [ ] Good job working on the desktop version 👍, but as I can see when you made the previous quire changes, now your `backound-image` link is broken. Kindly make sure you fixed it to replicate the design as much as possible (e.g., font, colors, images, text, margins) using [the info from the Figma templates](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md#:~:text=You%20must%20replicate%20the%20design%20as%20much%20as%20possible%20(e.g.%2C%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20info%20from%20the%20Figma%20templates) 😃.

### Figma

![Capture](https://user-images.githubusercontent.com/57408419/228786828-46661238-9504-4552-a150-a723041f7510.JPG)
![Capture](https://user-images.githubusercontent.com/57408419/228787630-385ecbb5-08b4-47dc-b474-ae85ff561916.JPG)


### Your project 

![Capture](https://user-images.githubusercontent.com/57408419/228787126-f1de337a-8dff-488c-857d-a71a17d92c7b.JPG)
![Capture](https://user-images.githubusercontent.com/57408419/228787322-98826fa2-e6b1-4415-b842-971520e02ba6.JPG)

Reply:
Please, make sure of it. I'm seeing it just fine. Look:
![image](https://user-images.githubusercontent.com/120040536/228792923-077fbf5e-553a-4ac1-b9e7-ad60359a277f.png)
![image](https://user-images.githubusercontent.com/120040536/228793063-2f4466f8-cc4b-4452-bde5-4278bd34699a.png)